app/settings/routes.py

```diff
@@ -31,6 +32,8 @@ def user_preferences():
             )
             current_user.store_history_record(record)
             db.session.commit()
+            # sends a notification email based on the attributes changed
+            send_notification_email(changes_made, current_user)
```

Comment:
Don't forget that we need to check the users notification settings and contact preference before calling this function.

Reply:
both checks have been added